js/grid/generate_grid.js

```diff
@@ -106,7 +111,12 @@ function appendTableData(trFall, trSpring, time) {
 
     var weekPrefixArray = ["M", "T", "W", "R", "F"];
 
-    var adjustedTime = (time === 12 ? 12 : time % 12) + ':00';
+    if (time % 1 == 0){
+        var adjustedTime = (time === 12 ? 12 : time % 12) + ':00';}
```

Comment:
The final brace, `}` should be located on the next line, beside the `else`. I've added a small example as to how this should look [here](https://github.com/Courseography/courseography/wiki/JavaScript-Code-Etiquette).